lua/ulx/modules/sh/cfc_crucify.lua

```diff
@@ -0,0 +1,146 @@
+CFCUlxCommands.crucify = CFCUlxCommands.crucify or {}
+local cmd = CFCUlxCommands.crucify
+
+CATEGORY_NAME = "Fun"
+
+function cmd.uncrucifyPlayer( ply )
+    if not IsValid( ply ) then return "Invalid player!" end
+
+    local cross = ply.cross
+    ply.cross = nil
+
+    local ragdoll = ply.ragdoll
+    if IsValid( ragdoll ) then
+        ulx.unragdollPlayer( ply )
+    end
+
+    if IsValid( cross ) then
+        cross:Remove()
+    end
+end
+
+local function crucifyPlayer( ply )
+    if not IsValid( ply ) then return "Invalid player!" end
+
+    ulx.ragdollPlayer( ply )
+    local playerRagdoll = ply.ragdoll
+
+    if not IsValid( playerRagdoll ) then return "Ragdoll failed" end
+
+    local cross = ents.Create( "prop_physics" )
+    cross:SetModel( "models/props_c17/gravestone_cross001a.mdl" )
+    cross:SetPos( ply:GetPos() )
+
+    local angles = ply:GetAngles()
+    cross:SetAngles( angles )
+
+    cross:Spawn()
+    cross:Activate()
+
+    local crossPhys = cross:GetPhysicsObject()
+    crossPhys:SetMass( 1000 ) -- make it easier to move the ragdoll attached to the cross
+    crossPhys:EnableMotion( false )
+
+    ply.cross = cross
+    cross.crucifiedPly = ply
+    playerRagdoll.crucifiedPly = ply
+
+    local phys_obj = playerRagdoll:GetPhysicsObjectNum( 1 )
+    local boneID = 2
+
+    while IsValid( phys_obj ) do
+        phys_obj = playerRagdoll:GetPhysicsObjectNum( boneID )
+
+        if phys_obj then
+            phys_obj:SetPos( phys_obj:GetPos() + Vector( 0, 0, 80 ) + angles:Forward() * 10 )
+        end
+        boneID = boneID + 1
+    end
+
+    local rightHandID = playerRagdoll:LookupBone( "ValveBiped.Bip01_R_Hand" )
+    local leftHandID = playerRagdoll:LookupBone( "ValveBiped.Bip01_L_Hand" )

```

Comment:
`Gets the bone index of the given bone name, returns nothing if the bone does not exist.`

These can error on models that don't have said bones, custom models from pac, outfitter w/e can cause this. Make sure to account for that.